package.json

```
@@ -16,6 +16,7 @@
 		"cli-link": "jetpack cli link",
 		"cli-setup": "pnpm install && jetpack cli link",
 		"cli-unlink": "jetpack cli unlink",
+		"component-usage-stats": "node ./node_modules/react-scanner/bin/react-scanner -c ./react-scanner.config.js",
```

Should be able to do this a bit more cleanly
```suggestion
		"component-usage-stats": "react-scanner -c ./react-scanner.config.js",
```